src/main/controllers/features/common/typeOfBusinessController.ts

```diff
@@ -34,24 +35,35 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
             });
         }
 
+        let typeOfBusiness = "";
+        // get data from mongo and save to session
+        try {
+            const acspData = await getAcspRegistration(session, session.getExtraData(SUBMISSION_ID)!, res.locals.userId);
+            saveDataInSession(req, USER_DATA, acspData);
+
+            if (acspData.typeOfBusiness === "UNINCORPORATED_ENTITY" || acspData.typeOfBusiness === "CORPORATE_BODY") {
```

Comment:
There is a chance of acsp data being null here as this is the first page... I think we should add that check as well.